samples/net/aws_iot/README.rst

```diff
@@ -152,7 +152,7 @@ The sample includes pre-configured configuration files for the development kits
 * :file:`boards/nrf9160dk_nrf9160_ns.conf` - Configuration file for the nRF9160 DK.
 * :file:`boards/thingy91_nrf9160_ns.conf` - Configuration file for the Thingy:91.
 * :file:`boards/nrf7002dk_nrf5340_cpuapp_ns.conf` - Configuration file for the nRF7002 DK.
-* :file:`boards/qemu_x86.conf` - Configuration file for QEMU x86.
+* :file:`boards/native_sim.conf` - Configuration file for NATIVE SIM.
```

Comment:
```suggestion
* :file:`boards/native_sim.conf` - Configuration file for the native simulator board.
```
This is the way the term is denoted in the zephyr docs: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/boards/native/native_sim/doc/index.html#native-sim